src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -57,7 +57,9 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction Char = _library.Add(new CharFunction());
         public static readonly TexlFunction CharT = _library.Add(new CharTFunction());
         public static readonly TexlFunction Clock24 = _library.Add(new IsClock24Function());
-        public static readonly TexlFunction Coalesce = _library.Add(new CoalesceFunction());
+        public static readonly TexlFunction Coalesce = _library.Add(new CoalesceFunction());
+        public static readonly TexlFunction Collect = _library.Add(new CollectScalarFunction());
+        public static readonly TexlFunction CollectT = _library.Add(new CollectFunction());
         public static readonly TexlFunction ColorFade = _library.Add(new ColorFadeFunction());
```

Comment:
We typically use the 'T' suffix for a tabular version of a scalar function - which is not the case here. Suggestion: Collect (for CollectFunction) and CollectS or CollectScalar (for CollectScalarFunction)